examples/list.birb

```diff
@@ -1,3 +1,10 @@
 List list = [10];
 
-scremLn(list[0]); // output 10
\ No newline at end of file
+scremLn(list[0]); // output 10
+scremLn(list.isEmpty); // output false
+scremLn(list.isNotEmpty); // output true
+scremLn(list.hashCode); // ignore
```

Comment:
You know the drill. Prefer validating sth you know about the value to ignoring it.

Reply:
Got it!